source/extensions/common/tap/admin.h

```diff
@@ -92,12 +92,7 @@ class AdminHandler : public Singleton::Instance,
     void bufferTrace(const std::shared_ptr<envoy::data::tap::v3::TraceWrapper>& trace);
 
     // Returns true if the trace buffer is full (reached max_buf_size_) false otherwise
-    bool full() const {
-      if (!buffer_) {
-        return false;
-      }
-      return (buffer_->size() == max_buf_size_);
-    }
+    bool full() const { return (buffer_ && buffer_->size() == max_buf_size_); }
```

Comment:
```suggestion
    bool full() const { return buffer_ && (buffer_->size() == max_buf_size_); }
```

I may be wrong and this doesn't match the style-guide (a bit more readable IMHO).

Reply:
BTW: is this change related to the PR, or just a fly-by change?